PyInstaller/building/datastruct.py

```diff
@@ -50,7 +50,7 @@ def _normentry(self, entry):
             logger.info("TOC found a %s, not a tuple", entry)
             raise TypeError("Expected tuple, not %s." % type(entry).__name__)
         name, path, typecode = entry
-        if typecode in ["BINARY", "DATA"]:
+        if typecode in ["BINARY", "DATA"] and not name.endswith('.typelib'):
             # Normalize the case for binary files and data files only (to avoid duplicates
```

Comment:
On Posix, `os.path.normcase` has no effect, and on WIndows, case does not matter when accessing the file. So why can't gobject-introspection find it?